Resources/Prototypes/_NF/Entities/Structures/Machines/Computers/mothership-computers.yml

```diff
@@ -0,0 +1,23 @@
+# The empress console
+- type: entity
+  id: EmpressMothershipComputer
+  name: Empress mothership console
+  parent: BaseMothershipComputer
+  components:
+  - type: Sprite
+    sprite: _NF/Structures/Machines/computers.rsi
+    layers:
+    - map: ["computerLayerBody"]
+      state: computer
+    - map: ["computerLayerKeyboard"]
+      state: generic_keyboard
+    - map: ["computerLayerScreen"]
+      state: shipyard_security
+    - map: ["computerLayerKeys"]
+      state: telesci_key
+  - type: ShipyardListing
+    name: Empress
+    shuttles:
```

Comment:
what exactly is being listed here? VesselPrototype IDs? should leave a comment either here or on the base yml prototype entry that describes exactly which ID we need to list since ships require several prototypes to function (vessel entry for the shipyard, station proto to function as stations, map proto to work with that as well)

Reply:
Yes, it's the vessel prototype id